src/calliope/example_models/urban_scale/model.yaml

```diff
@@ -19,10 +22,13 @@ config:
 
   solve:
     solver: cbc
+# --8<-- [end:config]
 
+# --8<-- [start:parameters]
 parameters:
   objective_cost_weights:
     data: 1
     index: monetary
     dims: costs
-  bigM: 1e6 # setting the scale of unmet demand, which cannot be too high, otherwise the optimisation will not converge
\ No newline at end of file
+  bigM: 1e6 # setting the scale of unmet demand, which cannot be too high, otherwise the optimisation will not converge
```

Comment:
```suggestion
  # `bigM` sets the scale of unmet demand, which cannot be too high,
  # otherwise the optimisation will not converge
  bigM: 1e6
```